exercises/E-conditionals/exercise.js

```diff
@@ -8,6 +8,11 @@
 
 let name = "Daniel";
 let danielsRole = "mentor";
+if (danielsRole === "mentor")
```

Comment:
Make sure to use curly braces to denote code blocks after `if/else` e.g. 
```
if (X) {
...
}
```

As otherwise JS will only execute the first like of code after the statement (which is why it works here, as you're only executing 1 line)

Reply:
Ok. Thank you. I will try to use but in Iran they taught me to use curly braces if you have more than one statement even for loop